sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -144,10 +143,14 @@ case class UnresolvedAttribute(nameParts: Seq[String]) extends Attribute with Un
   def name: String =
     nameParts.map(n => if (n.contains(".")) s"`$n`" else n).mkString(".")
 
-  override def exprId: ExprId = throw new UnresolvedException("exprId")
-  override def dataType: DataType = throw new UnresolvedException("dataType")
-  override def nullable: Boolean = throw new UnresolvedException("nullable")
-  override def qualifier: Seq[String] = throw new UnresolvedException("qualifier")
+  override def exprId: ExprId =
+    throw QueryCompilationErrors.invalidCallFunctionOnUnresolvedObjectError("exprId")
```

Comment:
Instead of using `QueryCompilationErrors.` multiple times, we can simplify this by importing `QueryCompilationErrors. invalidCallFunctionOnUnresolvedObjectError`.